js/flows/prompts/userPreference.v2.prompt

```diff
@@ -12,6 +12,8 @@ config:
       threshold: BLOCK_NONE
 input:
   schema: UserPreferenceFlowInputSchema
+output:
```

Comment:
In this one we want the second prompt to be the bad one and the first to be the good one. In the script we say that we have rolled out this second version and are testing it out. This represents a regression that we want folks to roll back.

Reply:
See the comments in the code where we instantiate the prompt.